core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -1,174 +1,354 @@
+    <h3 ng-if="$ctrl.isWindowNarrow()" translate="I18N_SPLASH_BENEFITS_TITLE"></h3>
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/matthew_fractions.png')]>" class="splash-matthew-image" alt="">
+  </div>
+  <div class="oppia-splash-section-three-right">
+    <h3 ng-if="!$ctrl.isWindowNarrow()" translate="I18N_SPLASH_BENEFITS_TITLE"></h3>
+    <div class="splash-benefits-section">
+      <img ng-src="<[$ctrl.getStaticImageUrl('/splash/bullet1.svg')]>" alt="">
+      <h2 translate="I18N_SPLASH_BENEFITS_ONE" class="splash-benefits"></h2>
     </div>
+    <div class="splash-benefits-section">
+      <img ng-src="<[$ctrl.getStaticImageUrl('/splash/bullet2.svg')]>" alt="">
+      <h2 translate="I18N_SPLASH_BENEFITS_TWO" class="splash-benefits"></h2>
+    </div>
+    <div class="splash-benefits-section">
+      <img ng-src="<[$ctrl.getStaticImageUrl('/splash/bullet3.svg')]>" alt="">
+      <h2 translate="I18N_SPLASH_BENEFITS_THREE" class="splash-benefits"></h2>
+    </div>
+  </div>
+</div>
 
-    <div class="oppia-splash-bullet">
-      <div class="oppia-splash-bullet-block oppia-splash-block-right-image">
-        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/bullet2icon.svg')]>" alt="">
-      </div>
-      <div class="oppia-splash-bullet-block oppia-splash-block-left-text"
-           translate="I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION"
-           dir="auto">
-      </div>
+<div class="oppia-splash-section-four">
+  <div class="oppia-splash-section-four-image-mobile" ng-if="$ctrl.isWindowNarrow()">
+    <img class="splash-student-desk-image" ng-src="<[$ctrl.getStaticImageUrl('/splash/student_desk_mobile.png')]>" alt="">
+  </div>
+  <div class="oppia-splash-section-four-left">
+    <h3 translate="I18N_SPLASH_FOR_STUDENTS"></h3>
+    <h2 translate="I18N_SPLASH_STUDENTS_TITLE"></h2>
+    <p translate="I18N_SPLASH_STUDENTS_CONTENT"></p>
+    <div class="oppia-splash-button-container">
+      <form action="<[$ctrl.classroomUrl]>">
+        <button type="submit"
+                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200"
+                translate="I18N_SPLASH_LEARN_MORE">
+        </button>
+      </form>
     </div>
+  </div>
+  <div class="oppia-splash-section-four-right" ng-if="!$ctrl.isWindowNarrow()">
+    <img class="splash-student-desk-image" ng-src="<[$ctrl.getStaticImageUrl('/splash/student_desk.png')]>" alt="">
+  </div>
+</div>
 
-    <div class="oppia-splash-bullet">
-      <div class="oppia-splash-bullet-block oppia-splash-block-left-image">
-        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/bullet3icon.svg')]>" alt="">
-      </div>
-      <div class="oppia-splash-bullet-block oppia-splash-block-right-text"
-           translate="I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION"
-           dir="auto">
-      </div>
+<div class="oppia-splash-section-five" ng-style="{'background-image': '<[$ctrl.getTestimonialBackgroundUrl()]>'}">
```

Comment:
Could this section be generalized so that instead of checking for `$ctrl.testimonialId === x` we would request the required data from some dictionary? I think this approach is used on the landing page.

Reply:
Done